packages/orchestrator-ui-components/src/pages/processes/WfoStartProcessPage.tsx

```diff
@@ -133,13 +137,12 @@ export const WfoStartProcessPage = ({
                         throw e;
                     },
                 )
-                .catch((error: AxiosError) => {
-                    if (error?.response?.status !== 510) {
-                        if (error?.response?.status === 400) {
+                .catch((error) => {
+                    if (error?.status !== 510) {
```

Comment:
Can you use the HttpStatus enum here aswell, and other places we check status codes? I do prefer to keep this checking consitent. So we choose on or the other